### PR TITLE
Feat user role

### PR DESCRIPTION
**Pull request by shaarful**
## Description
Add new patch to main Kobo container

- [x] I have read the road86 Contribution Guide.
- [x] I have checked all commit message styles match the requested structure.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have assigned at least one reviewer.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] My changes generate no new warnings.
- [x] I have made corresponding changes to the documentation.
- [x] New database changes have been committed.

### CODE REVIEW

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]
Again, looks like something has gone wrong with the rebasing here?
Commit [138637a](https://github.com/road86/bahis-infra/pull/33/commits/138637ae002654cf46b3ddf85d4548290191cc71) actually seems to include stuff for the dashboard and docker networking not just user roles in kobo?

**Comment by shaarful** [comment]
> Again, looks like something has gone wrong with the rebasing here? Commit [138637a](https://github.com/road86/bahis-infra/pull/33/commits/138637ae002654cf46b3ddf85d4548290191cc71) actually seems to include stuff for the dashboard and docker networking not just user roles in kobo?

In this particular commit, I add dash-ulo repo for BAHIS 3. Somehow I can't see `dash ulo` git module in `bahis3` branch.

File: bahis-kobo/kpi/settings/dev.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import os
+from kobo.settings.dev import *
+
+
+SESSION_COOKIE_NAME = 'sharful'
+SESSION_COOKIE_DOMAIN = '.sharful.com'
```
Comment (ChasNelson1990) [inline bahis-kobo/kpi/settings/dev.py]: This looks wrong?

**Review by ChasNelson1990 — COMMENTED** [review]
Everything seems set up for your personal site?

File: bahis-kobo/kpi/settings/dev.py (hunk 1 above)
Comment (shaarful) [reply bahis-kobo/kpi/settings/dev.py]: > This looks wrong?

yes. it should be in the environment file. I am making the change. thanks

Comment (shaarful) [reply bahis-kobo/kpi/settings/dev.py]: @ChasNelson1990 Fixed

**Comment by shaarful** [comment]
> Everything seems set up for your personal site?

I have added new patch on kpi to set extra settings and URLs.

**Review by ChasNelson1990 — APPROVED** [review]
Lots in here, but looks good to me

**Pull request merged**